components/Login/SignIn.tsx

```diff
@@ -32,26 +36,57 @@ interface FormValues {
   checked: Array<string>
 }
 
+// checks if the user is able to log in
+export function login(data: Session | null, router: NextRouter) {
+  if (data?.user) {
+    if (data.user.id) {
+      router.push('/DynamicPOC')
+    } else {
+      alert(data.user.name)
+    }
+  }
+}
+
 // component for the signin page - includes form (and styling) for validation
 function SignIn() {
+  const router = useRouter()
+  const { data } = useSession()
   const initialVal: FormValues = {
     email: '',
     password: '',
     checked: [],
   }
 
+  // check if the user is invalid
+  useEffect(() => {
+    if (data?.user) {
+      if (data.user.id) {
+        router.push('/DynamicPOC')
+      } else {
+        alert(data.user.name)
+      }
+    }
```

Comment:
delete this use effect (it's the same as the one below)

Reply:
oops lol